### ISSUE TITLE
return correct status codes when creating new resources

### ISSUE THREAD

**Issue by Toerktumlare** [Ayushnita/Resume_builder#4]
rfc7231 states the following when creating a new resources (POSTing to your application)

> The 201 (Created) status code indicates that the request has been
   fulfilled and has resulted in one or more new resources being
   created.  The primary resource created by the request is identified
   by either a Location header field in the response or, if no Location
   field is received, by the effective request URI.

[https://tools.ietf.org/html/rfc7231#section-6.3.2](https://tools.ietf.org/html/rfc7231#section-6.3.2)

So basically returning a Http status of 201 Created and not 200 OK and the location of the new resource in the `Location` Header.
